clients/spark/LICENSE

```diff
@@ -1,201 +0,0 @@
-                                 Apache License
```

Comment:
@ozkatz do we need to keep this or can we remove?  It's a separate component, inside the same repo...

Reply:
Got the LICENSE file back